### DISCUSSION TITLE
Technical issue

### DISCUSSION THREAD

**Discussion by Shikiiii** [community/community#81493] [GitHub Education]
### Hi everyone,

Hello dear GitHub Community.

About a week ago my application was rejected. That's fine! GitHub now wants me to take a photo of required documents using their built-in camera into the website! That is also fine, until it isn't. No matter what I do, I cannot switch to the back camera. I see 3 options, both of them showing my front camera. I would express how difficult it is to take a readable picture of a document using your front camera, but you can imagine that yourself. Has anyone ran into similar issues and/or does anyone know how to fix this? P.S. I don't have a camera on my PC, otherwise I would've done it that way.

Thanks in advance.

**Comment by BilalBoudjema** [comment]
# Bug Report: GitHub Discussions Link Behavior

## Description
Encountering an issue in GitHub Discussions where the organization website link opens in the same tab instead of a new one, despite an icon indicating it should open in a separate tab.

## Steps to Address the Issue

### 1. Report the Bug
- Create an issue in the [GitHub Feedback discussions](https://github.com/github/feedback/discussions/categories/discussions).
- Provide a clear outline of the issue, steps to reproduce, and relevant screenshots.

### 2. Check for Updates
- Ensure your browser and any GitHub-related extensions are up-to-date.

### 3. Alternative Workaround
- Consider advising users to right-click and open the link in a new tab as a temporary solution.

### 4. Consult GitHub Documentation
- Review GitHub's Discussions documentation for any relevant settings or configurations.

### 5. Seek Community Input
- Post the issue in GitHub community forums for insights or solutions from other users.

### 6. Monitor GitHub Updates
- Stay updated with GitHub’s releases for any fixes or updates regarding this issue.

### 7. Contact GitHub Support
- If the issue persists, contact GitHub Support directly for assistance.

## Additional Notes

- Provide detailed information when reporting to help GitHub address the problem efficiently.
